package.json

```diff
@@ -37,12 +36,10 @@
   },
   "lint-staged": {
     "*.{js}": [
-      "eslint --fix --quiet",
-      "git add"
+      "eslint --fix --quiet"
     ],
     "*.{js,json}": [
-      "prettier --write",
-      "git add"
```

Comment:
newer version of `lint-staged` handles `git add` for us, so the docs said to remove it.